HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift

```diff
@@ -11,7 +11,7 @@ import UIKit
  
  */
 public func giniHealthBundle() -> Bundle {
-    Bundle.module
+    giniHealthBundleResource()
```

Comment:
Regarding: `giniHealthBundle()`
I would rather leave that as it is and I would use `giniHealthBundleResource()` in the `defaultImage()` function. I see there is the only place where is used.